model/models.go

```diff
@@ -61,10 +67,30 @@ type Offering struct {
 	CreatedBy   string    `json:"created_by"`
 }
 
+func (this *Offering) Equal(other *Offering) bool {
+	return this.ID == other.ID && this.WorkspaceID == other.WorkspaceID &&
+		this.UserID == other.UserID && this.StartDate.Equal(other.StartDate) &&
+		this.EndDate == other.EndDate && this.Cancelled == other.Cancelled && this.CreatedBy == other.CreatedBy
+}
+
+func (this *Offering) EqualMinusID(other *Offering) bool { // To be used when testing Creation, as ID will not be known in advance.
```

Comment:
i like the `Equal` function but this function might be kinda overkill. If u r only testing certain field then u should use a conditional instead of writing a public function (consider a helper in the test instead)
I would also rather have u use `asserts` in this case
```golang
assert.Equal(t, this.ID, other.ID)
assert.Equal(t, this.WorkspaceID, other.WorkspaceID)
...
```